apps/condo/jest.setup.js

```diff
@@ -10,6 +10,7 @@ if (conf.TESTS_FAKE_WORKER_MODE) createWorker(require.resolve('./index')).catch(
 })
 
 beforeAll(async () => {
+    jest.setTimeout(10000000)
```

Comment:
What's this line doing?

Reply:
It removes the default jest timeout for async tests (30 sec one)

Reply:
I made it more clear